javascript/ql/src/semmle/javascript/dataflow/Nodes.qll

```diff
@@ -814,6 +881,9 @@ module ClassNode {
       kind = MemberKind::method() and
       result = getAPrototypeReference().getAPropertyWrite().getRhs().getALocalSource()
       or
+      kind = MemberKind::method() and
+      result = getConstructor().getReceiver().getAPropertyWrite().getRhs().getALocalSource()
```

Comment:
Ditto here (and probably elsewhere as well).

Reply:
Hm, there is no 0-argument version of `getAPropertySource()`. Should we just add it now as a drive-by change (and hope it doesn't wobble the optimizer)?

Reply:
Good idea.